tests/unit/utils/test_networking_utils.py

```diff
@@ -0,0 +1,93 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+import socket
+
+import hypothesis.strategies as st
+from hypothesis import given
+
+from zenml.utils import networking_utils
+
+
+def find_open_port():
+    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    s.bind(("", 0))
+    s.listen(1)
+    port = s.getsockname()[1]
+    s.close()
+    return port
+
+
+def test_port_available_works():
+    """Test if port_available works."""
+    open_port = find_open_port()
+    networking_utils.port_available(open_port) is True
```

Comment:
```suggestion
    assert networking_utils.port_available(open_port) is True
```

... is probably what you meant.